x-pack/plugins/security_solution/public/management/components/artifact_entry_card/components/card_decorators/event_filters_process_descendant_indicator.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+import React, { memo } from 'react';
+import { EuiSpacer, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useIsExperimentalFeatureEnabled } from '../../../../../common/hooks/use_experimental_features';
+import { isFilterProcessDescendantsEnabled } from '../../../../../../common/endpoint/service/artifacts/utils';
+import { ProcessDescendantsTooltip } from '../../../../pages/event_filters/view/components/process_descendant_tooltip';
+import type { ArtifactEntryCardDecoratorProps } from '../../artifact_entry_card';
+
+export const EventFiltersProcessDescendantIndicator = memo<ArtifactEntryCardDecoratorProps>(
+  ({ item, testIdPrefix }) => {
+    const isProcessDescendantFeatureEnabled = useIsExperimentalFeatureEnabled(
+      'filterProcessDescendantsForEventFiltersEnabled'
+    );
+
+    if (
+      isProcessDescendantFeatureEnabled &&
+      isFilterProcessDescendantsEnabled(item as ExceptionListItemSchema)
+    ) {
+      return (
+        <>
+          <EuiText data-test-subj={testIdPrefix && `${testIdPrefix}-processDescendantIndication`}>
+            <code>
+              <strong>
+                <FormattedMessage
+                  defaultMessage="Filtering descendants of process"
+                  id="xpack.securitySolution.eventFilters.filteringProcessDescendants"
+                />{' '}
+                <ProcessDescendantsTooltip
+                  indicateExtraEntry
+                  testIdPrefix={testIdPrefix && `${testIdPrefix}-processDescendantIndication`}
```

Comment:
I wonder if this wouldn't end up being `false` if there is no prefix.

Reply:
`testIdPrefix` is either a `string` or `undefined` and the `&&` should stop processing operands and return the current operand as soon as it is falsy (or the last one if all are truthy), so if
- the prefix is a truthy string, the result will be `somestring-processDescendantIndication`
- if an empty string, the result will be `` also empty - not ideal :)
- if `undefined`, the result also will be `undefined`

so it cannot be `false`, but it's not good nevertheless, i'll change it to
```ts
testIdPrefix !== undefined ? `${testIdPrefix}-processDescendantIndication` : undefined
```
thanks :)

Reply:
d8990d566645d801f3af06f9500bc6df63ded15d